Explorer/Assets/DCL/Multiplayer/Connections/Rooms/Connective/IConnectiveRoom.cs

```diff
@@ -1,55 +1,25 @@
 using Cysharp.Threading.Tasks;
-using LiveKit.Rooms;
+using System.Threading;
 
 namespace DCL.Multiplayer.Connections.Rooms.Connective
 {
+    /// <summary>
+    ///     This interface became redundant but I keep it if we want to mock in the future
+    /// </summary>
```

Comment:
It's impossible to mock, everywhere is hardcoded a current implementation as the dependency

Reply:
I changed to `IConnectiveRoom`